### ISSUE TITLE
Increase max input tokens for puter-chat-completion from 4k to 128k

### ISSUE THREAD

**Issue by recursionbane** [HeyPuter/puter#773]
Hi, Puter currently [hardcodes](https://github.com/HeyPuter/puter/blob/main/src/backend/src/modules/puterai/OpenAICompletionService.js#L195) the max input tokens for any chat completion request to 4k, but the default model (gpt-4o-mini) supports [128k](https://platform.openai.com/docs/models/gpt-4o-mini) input tokens.

Can we increase the max token limit to 128k to match the token limit for gpt-4o-mini?

Thanks in advance!

**Comment by jelveh** [comment]
Yes, we should be able to do this. I need to do the math for costs. I'll update this again soon
